pwiz_tools/Skyline/ToolsUI/PythonInstallerDlg.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Diagnostics;
+using System.Drawing;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using pwiz.Common.Collections;
+using pwiz.Skyline.Alerts;
+using pwiz.Skyline.Controls;
+using pwiz.Skyline.Model.Tools;
+using pwiz.Skyline.Util;
+using pwiz.Skyline.Util.Extensions;
+
+namespace pwiz.Skyline.ToolsUI
+{
+    public partial class PythonInstallerDlg : FormEx
```

Comment:
We should just use MessageDlg or MultiButtonMessageDlg instead of this custom form. All it does now is show a message and buttons. There are other good examples of this for installation.